Sources/StytchCore/WebAuthenticationSessionClient/WebAuthenticationSessionClient.swift

```diff
@@ -25,6 +25,7 @@ extension WebAuthenticationSessionClient {
         #if !os(tvOS)
         let presentationContextProvider: ASWebAuthenticationPresentationContextProviding
         #endif
+        let clientType: ClientType
```

Comment:
I think this is probably what you were going for before and having trouble with, but you could probably have this type be `StytchClientType.Type` and then at the call site just pass in `StytchClient.self` or `StytchB2BClient.self`. I _think_ that would work.